web_autoflip_demo/web_worker_pm/main.ts

```diff
@@ -0,0 +1,52 @@
+/**
+Copyright 2020 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+declare let buffer: ArrayBuffer;
+
+/** Read input video as an array buffer */
```

Comment:
Thanks for the function comment changes! One last thing is to imagine before the comment exists an implict "This method" "YOUR COMMENT" so make convert "Read input video as an array buffer" to (implicit "This method") "Read**s** input video as an array buffer" (add s after Read)

Reply:
Sure, thank you so much for the detailed explanation.